.k8/hetzner-stage/ssv-node-44-deployment.yml

```diff
@@ -48,86 +48,90 @@ spec:
-          subPath: share.yaml
-          name: ssv-node-44-cm
+        - name: ssv-node-44
+          image: REPLACE_DOCKER_REPO:REPLACE_IMAGETAG
+          #image: mosheblox/ssv-preview:stage
+          imagePullPolicy: Always
+          resources:
+            limits:
+              cpu: REPLACE_NODES_CPU_LIMIT
+              memory: REPLACE_NODES_MEM_LIMIT
+          command: ["make", "start-node"]
+          ports:
+            - containerPort: 12044
+              name: port-12044
+              protocol: UDP
+              hostPort: 12044
+            - containerPort: 13044
+              name: port-13044
+              hostPort: 13044
+            - containerPort: 15044
+              name: port-15044
+              hostPort: 15044
+            - containerPort: 16044
+              name: port-16044
+              hostPort: 16044
+          env:
+            - name: SHARE_CONFIG
+              value: "./data/share.yaml"
+            - name: CONFIG_PATH
+              valueFrom:
+                secretKeyRef:
+                  name: config-secrets
+                  key: config_path
+            - name: ABI_VERSION
+              valueFrom:
+                secretKeyRef:
+                  name: config-secrets
+                  key: abi_version
+                  optional: true
+            - name: LOG_LEVEL
+              value: "debug"
+            - name: DEBUG_SERVICES
+              value: "ssv/*."
+            - name: DISCOVERY_TYPE_KEY
+              value: "discv5"
+            - name: CONSENSUS_TYPE
+              value: "validation"
+            - name: HOST_DNS
+              value: ""
+            - name: HOST_ADDRESS
+              value: ""
+            - name: DB_PATH
+              value: "./data/db-holesky-stage"
+            - name: NETWORK
+              value: "holesky-stage"
+            - name: DB_REPORTING
+              value: "false"
+            - name: METRICS_API_PORT
+              value: "15044"
+            - name: SSV_API_PORT
+              value: "16044"
+            - name: ENABLE_PROFILE
+              value: "true"
+            - name: DISCOVERY_TRACE
+              value: 'false'
+            - name: PUBSUB_TRACE
+              value: 'false'
+            - name: BUILDER_PROPOSALS
+              value: "true"
+          volumeMounts:
+            - mountPath: /data
+              name: ssv-node-44
+            - mountPath: /data/share.yaml
+              subPath: share.yaml
+              name: ssv-node-44-cm
       imagePullSecrets:
         - name: ecr-repo
       volumes:
-      - name: ssv-node-44
-        persistentVolumeClaim:
-          claimName: ssv-node-44
-      - name: ssv-node-44-cm
-        configMap:
-          name: ssv-node-44-cm
+        - name: ssv-node-44
+          persistentVolumeClaim:
+            claimName: ssv-node-44
+        - name: ssv-node-44-cm
+          configMap:
+            name: ssv-node-44-cm
       hostNetwork: true
+  ports:
+    - null
+    - null
```

Comment:
care to explain the `null`s?

Reply:
Ah nice catch, wrote a script and didn't realize there were 2 specs per file. Should be good now.